Modules/Sources/JetpackStats/Charts/BarChartView.swift

```diff
@@ -220,6 +222,97 @@ struct BarChartView: View {
             )
         }
     }
+
+    // MARK: - Gestures
+
+    private func makeGesturesOverlayView(proxy: ChartProxy) -> some View {
+        GeometryReader { geometry in
```

Comment:
This turned out to be more complicated that I wanted, but at least it's isolated to this component. The rest of the implementation is pretty straightforward and doesn't affect the overall architecture.